specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/preview/2017-11-03-preview/hanaonazure.json

```diff
@@ -1174,6 +1174,10 @@
           "type": "string",
           "description": "MSI ID passed by customer which has access to customer's KeyVault and to be assigned to the Collector VM."
         },
+        "keyVaultId": {
```

Comment:
Is this read only property? Typically, settable properties must be added in a new api-version. but since this is preview, we can evaluate the customer impact and then decide. What happens on the subsequent PUT which doesnt contain this property in the request body and if the first PUT had set it to some value?

Reply:
This is a new API and no customer is using it yet.

As for the PUT, we only support create and delete, not update. If you call PUT on this API, it will error out saying the resource is already created.